packages/round-manager/src/context/round/CreateRoundContext.tsx

```diff
@@ -139,10 +145,14 @@ const _createRound = async ({
   try {
     datadogLogs.logger.info(`_createRound: ${round}`);
 
-    if (roundMetadataWithProgramContractAddress && roundMetadataWithProgramContractAddress.eligibility) {
-      roundMetadataWithProgramContractAddress.eligibility.requirements = roundMetadataWithProgramContractAddress.eligibility?.requirements.filter(
-        obj => obj.requirement != ""
-      );
+    if (
+      roundMetadataWithProgramContractAddress &&
+      roundMetadataWithProgramContractAddress.eligibility
+    ) {
+      roundMetadataWithProgramContractAddress.eligibility.requirements =
+        roundMetadataWithProgramContractAddress.eligibility?.requirements.filter(
+          (obj) => obj.requirement != ""
```

Comment:
I just formatted this code, didn't write it. Not sure why the original author used loose comparison here, but I'm too scared to make it strict because not sure if the requirements are actually just empty strings, or if they can be nulls etc. So would leave as-is for now.

Reply:
Agreed. Can you just add a comment so anybody who's tempted to "fix" knows why it's like that?